ivy/functional/frontends/numpy/random/functions.py

```diff
@@ -162,3 +162,9 @@ def negative_binomial(n, p, size=None):
         size = (size,)
     lambda_ = ivy.gamma(n, scale, shape=size)
     return ivy.poisson(lam=lambda_, shape=size)
+
+@to_ivy_arrays_and_back
+@from_zero_dim_arrays_to_scalar
+def laplace_sample(mean=0.0, scale=1.0, size=None):
```

Comment:
Normally the name of the front-end function should be the same as that of the original library. In numpy, I think this is `laplace`.